loadtest.py

```diff
@@ -39,10 +39,10 @@ def logout():
     assert res.status < 400
 
 
-@scenario(WEIGHT_READ_SHOT)
-async def read_shot(session):
-    shot = await utils.create_shot(session)
-    assert shot.status < 400
+# @scenario(WEIGHT_READ_SHOT)
+# async def read_shot(session):
+#     shot = await utils.create_shot(session)
+#     assert shot.status < 400
 
-    res = await utils.read_shot(session, shot.path)
-    assert res.status < 400
+#     res = await utils.read_shot(session, shot.path)
+#     assert res.status < 400
```

Comment:
I don't think we're currently using this test case. Instead we're using the `loadtest-read.py#read_shot()` scenario since it just uploads a single shot, and then reads it infinity number of times (letting us measure the pure read_shot performance -- versus this test which muddies it up with a multi-test of `create_shot+read_shot`).